properties/P000076.md

```diff
@@ -25,10 +27,13 @@ or the intersection $\bigcap_{n<\omega}D_n[x_n]$ is empty.
 
 Note that while many authors require {P76} spaces to be {P3}
 (equivalently, {P1}),
-we do not require this separation axiom here.
+we do not require this separation axiom here; note the meta-property below
+that shows this property holds exactly when it holds for the Kolmogorov quotient.
```

Comment:
```suggestion
we do not require this separation axiom here.
But by one of meta-properties below, this property holds exactly when it holds for the Kolmogorov quotient,
which is {P3}.
```
added mention of Hausdorff, which I assume is the reason you added this sentence here?

Reply:
```suggestion
we do not require this separation axiom here.
But by one of meta-properties below, this property holds exactly when it holds for its Kolmogorov 
({P1}) quotient (which must in fact be {P3}, since all {P76} spaces are {P12}).
```